src/pages/how-to/delete-account.mdx

```diff
@@ -0,0 +1,32 @@
+import {Note} from "@/components/mdx";
+
+# Delete your NetBird account
+To delete your NetBird account, you must be a user with the Owner role. You can ask the owner to delete your account if you are not the account owner.
```

Comment:
`You can ask the owner to delete your account if you are not the account owner.
`
I think that this is misleading, becaue it might be mixed up with the user accounts.

Reply:
We should also add information about deleting all associated information and that this actioan will delete every user, peer, etc...

Reply:
Need t oadd reference to the user roles doc
https://github.com/netbirdio/docs/pull/120